docs/20_smart-contracts/10_getting-started/10_dune-guide/index.md

```diff
@@ -0,0 +1,634 @@
+
+permission links:
+     eosio.any:
+
+memory:
+     quota:       unlimited  used:      2.66 KiB
+
+net bandwidth:
+     used:               unlimited
+     available:          unlimited
+     limit:              unlimited
+
+cpu bandwidth:
+     used:               unlimited
+     available:          unlimited
+     limit:              unlimited
+
+subjective cpu bandwidth:
+     used:                 0 us
+```
+
+Notice the account's creation date and time, the default permissions `owner` and `active`, their corresponding public keys, and default values for RAM, NET, and CPU resources. For more information about EOS resources, visit the [Resources guide](../30_resources/index.md) guide.
+
+Finally, create two test accounts, `bob` and `alice` by executing the following DUNE commands:
+
+```shell
+$ dune --create-account bob
+$ dune --create-account alice
+```
+
+Now that your contract account has been created, you can build, deploy, and test your smart contract in the next section.
+
+## Smart Contract Development
+
+DUNE provides at least two commands to create smart contract projects:
+* `create-bare-app`, for simple projects, invoking `cdt` directly
+* `create-cmake-app`, for larger projects, using `cmake` builds
+
+Although a bare smart contract project makes more sense for single contract projects, for this guide you will create a cmake project that you will expand. For larger projects involving multiple contracts and/or web applications, a cmake project makes more sense.
+
+## Hello World Contract
+
+These are the steps that you need to follow to create your "Hello World" smart contract:
+
+[Step 1](#step-1---create-project) - Create new cmake smart contract project `hello`
+[Step 2](#step-2---write-contract) - Write the `hello` smart contract
+[Step 3](#step-3---compile-contract) - Compile the `hello` contract
+[Step 4](#step-4---deploy-contract) - Deploy the `hello` contract
+
+### Step 1 - Create Project
+
+Create the cmake smart contract project `hello` in the current directory by running the following command:
+
+```shell
+dune --create-cmake-app hello .
+```
+
+The command will create the following file structure:
+
+```shell
+ls -l hello
+```
+```
+-rw-r--r-- 1 lupaxel lupaxel  442 Feb 17 10:20 CMakeLists.txt
+-rw-r--r-- 1 lupaxel lupaxel  462 Feb 17 10:20 README.txt
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 build
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 include
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 ricardian
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 src
+```
+
+```shell
+ls -l hello/src
+```
+```
+-rw-r--r-- 1 lupaxel lupaxel 215 Feb 17 10:20 CMakeLists.txt
+-rw-r--r-- 1 lupaxel lupaxel 110 Feb 17 10:20 hello.cpp
+```
+
+Now you will modify the source file `hello.cpp` to implement your contract.
+
+### Step 2 - Write Contract
+
+Enter the `hello` project main directory:
+
+```shell
+cd hello
+```
+
+Open the `hello` project folder in your code editor or IDE of choice:
+
+```shell
```

Comment:
Isn't `code .` only for VS studio?

If so, I don't mind us telling people to use VS Studio. It's free and we'll have one target to support.

Reply:
Sounds good, mentioned VS Code